classes/controller/AdminController.php

```diff
@@ -2614,6 +2614,7 @@ public function setMedia($isNewTheme = false)
 
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/theme.css', 'all', 0);
             $this->addJS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/main.bundle.js');
+            $this->addJS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/new-theme/public/create_product.bundle.js');
```

Comment:
You should add the dedicated CSS for default theme here `create_product_default_theme.bundle.css`